webknossos/webknossos/dataset/_utils/buffered_slice_writer.py

```diff
@@ -32,10 +33,13 @@ class BufferedSliceWriter:
     def __init__(
         self,
         view: "View",
-        offset: Vec3IntLike,
+        offset: Optional[Vec3IntLike] = None,
```

Comment:
The two classes `BufferedSliceWriter` and `BufferedSliceReader` are not considered as part of the API (also not part of the docs). I'll adapt the View's methods `get_buffered_slice_writer` and `_reader` though.

Reply:
> The two classes BufferedSliceWriter and BufferedSliceReader are not considered as part of the API (also not part of the docs).

Then, I'd suggest to remove the existing docstrings for the constructors. Otherwise, they would get out-of-sync with the constructors' arguments.